frontend/src/components/general/DatePicker.tsx

```diff
@@ -25,8 +27,8 @@ export default function DatePicker({
     handleChange(selectedDate);
   };
   return (
-    <MuiPickersUtilsProvider utils={DateFnsUtils}>
-      <KeyboardDatePicker
+    <LocalizationProvider dateAdapter={AdapterDateFns}>
+      <DatePickerComponent
```

Comment:
Did you test this change locally? Is it working everywhere we use date picker?

Reply:
Yes